projects/js-packages/critical-css-gen/tsconfig.node.json

```diff
@@ -0,0 +1,16 @@
+{
+	"extends": "jetpack-js-tools/tsconfig.tsc.json",
+	"include": [ "src/back-end.ts" ],
+	"exclude": [ "node_modules/**/*" ],
+	"compilerOptions": {
+		"outDir": "./lib",
+		"target": "es2019",
+		"sourceMap": true,
+		"allowJs": false,
+		"allowSyntheticDefaultImports": true,
+		"typeRoots": [ "node_modules/@types" ],
+		"declaration": true,
+		"module": "NodeNext",
+		"moduleResolution": "NodeNext"
```

Comment:
It's not clear to me why you even have two different tsconfig files. This escapes our usual linting check that says not to set `module` and `moduleResolution` directly.

And these are already the values set by `tsconfig.tsc.json` anyway, so even less need to have them here.

Reply:
I guess I got confused and thought that somehow rollup uses the default tsconfig.

Thanks for raising this, I've removed the `tsconfig.node.json`.